luigi_pipeline/lib/model/mito_mt_schema.py

```diff
@@ -0,0 +1,113 @@
+import hail as hl
+
+from lib.model.base_mt_schema import row_annotation
+from lib.model.seqr_mt_schema import BaseSeqrSchema, SeqrGenotypesSchema, SeqrVariantsAndGenotypesSchema
+
+
+class SeqrMitoVariantSchema(BaseSeqrSchema):
+
+    def __init__(self, *args, high_constraint_region, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._high_constraint_region = high_constraint_region
+
+    # Mitochondrial only fields
+    @row_annotation()
+    def gnomad_mito(self):
+        return self._selected_ref_data.gnomad_mito
+
+    @row_annotation()
+    def mitomap(self):
+        return self._selected_ref_data.mitomap
+
+    @row_annotation(name='APOGEE')
+    def mitimpact(self):
+        return self._selected_ref_data.mitimpact
+
+    @row_annotation(name='HmtVar')
+    def hmtvar(self):
+        return self._selected_ref_data.hmtvar
+
+    @row_annotation()
+    def helix(self):
+        return self._selected_ref_data.helix_mito
+
+    @row_annotation()
+    def common_low_heteroplasmy(self):
+        return self.mt.common_low_heteroplasmy
+
+    @row_annotation()
+    def hap_defining_variant(self):
+        return self.mt.hap_defining_variant
+
+    @row_annotation()
+    def mitotip_trna_prediction(self):
+        return self.mt.mitotip_trna_prediction
+
+    @row_annotation()
+    def high_constraint_region(self):
+        return hl.is_defined(self._high_constraint_region[self.mt.locus])
+
+    # Fields with the same names but annotated differently
+    @row_annotation()
+    def rsid(self):
+        return self.mt.rsid.find(lambda x: hl.is_defined(x))
+
+    @row_annotation(name='AC')
+    def ac(self):
+        return self.mt.AC_hom
+
+    @row_annotation(name='AF')
+    def af(self):
+        return self.mt.AF_hom
+
+    @row_annotation(name='AN')
+    def an(self):
+        return self.mt.AN
+
+    @row_annotation(name='AC_het')
+    def ac_het(self):
```

Comment:
these fields are mito specific, not "Fields with the same names but annotated differently" so you should move them above that comment